Lab_1.m

```diff
@@ -0,0 +1,94 @@
+%1 
+
+d = random('Normal', 0, 1, [1,100000]);
+histogram(d,50,'DisplayStyle',"stairs",'LineWidth',3)
+xlim([-5,5])
+
+histogram(d,50,'DisplayStyle',"stairs",'LineWidth',3,"Normalization","pdf")
+hold on 
+x = linspace(5.95,6.05,1000);
```

Comment:
Since you set mean value=0 and sigma value = 1 when you created random distribution (line 3), I think you should change range of x to -5<x<5 (i.e. x = linspace(-5,5,1000)).